types/electron-settings/index.d.ts

```diff
@@ -0,0 +1,257 @@
+// Type definitions for electron-settings 3.0
+// Project: https://github.com/nathanbuchar/electron-settings#readme
+// Definitions by: Ian Copp <https://github.com/icopp>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="node"/>
+
+import * as fs from "fs";
+
+type SettingsHandler = (newValue: any, oldValue: any) => any;
+
+interface JsonObject {
+    [x: string]: JsonValue;
+}
+
+type JsonValue = string | number | boolean | null | string[] | number[] | boolean[] | JsonObject;
+
+interface SettingsOptions {
+    /**
+     * Prettify the JSON output. Defaults to `false`.
+     */
+    prettify?: boolean;
+}
+
+declare class Settings extends NodeJS.EventEmitter {
```

Comment:
Would make more sense to just `declare interface Settings`, omit all private members,  and use `declare var settings: Settings; export = settings;`